backend/config/database.yml

```
@@ -20,6 +20,10 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV['USE_DOCKER'] == 'true' ? 'db' : 'localhost' %>
+  username: postgres
+  password: postgres
+  database: potter_db_api_development
```

```suggestion
  host: <%= ENV.fetch("DB_HOST") { "localhost" } %>
  username: <%= ENV.fetch("POSTGRES_USER") { "" } %>
  password: <%= ENV.fetch("POSTGRES_PASSWORD") { "" } %>
```
I would suggest doing it like this.

---

Nice!